po/es.po

```diff
@@ -0,0 +1,1212 @@
+
+#: src\just-rust.md:19
+msgid "So, we upstreamed all of them"
+msgstr "Pues los upstreameamos todos"
+
+#: src\just-rust.md:20
+msgid ""
+"Like [\\#93717](https://github.com/rust-lang/rust/pull/93717), "
+"[\\#108659](https://github.com/rust-lang/rust/pull/108659), "
+"[\\#111936](https://github.com/rust-lang/rust/pull/111936), "
+"[\\#108898](https://github.com/rust-lang/rust/pull/108898)..."
+msgstr ""
+"Como [\\#93717](https://github.com/rust-lang/rust/pull/93717), "
+"[\\#108659](https://github.com/rust-lang/rust/pull/108659), "
+"[\\#111936](https://github.com/rust-lang/rust/pull/111936), "
+"[\\#108898](https://github.com/rust-lang/rust/pull/108898)..."
+
+#: src\just-rust.md:21
+msgid ""
+"[\\#111992](https://github.com/rust-lang/rust/pull/111992), "
+"[\\#112314](https://github.com/rust-lang/rust/pull/112314), "
+"[\\#112418](https://github.com/rust-lang/rust/pull/112418), "
+"[\\#112454](https://github.com/rust-lang/rust/pull/112454), ..."
+msgstr ""
+
+#: src\just-rust.md:32
+msgid "Virtuous Cycle"
+msgstr "Ciclo Virtuoso"
+
+#: src\just-rust.md:34
+msgid "Sometimes we find bugs that upstream missed"
+msgstr "A veces encontramos bugs que perdió upstream"
+
+#: src\just-rust.md:35
+msgid "So we upstreamed the fixes"
+msgstr "Y les upstreameamos los fixes"
+
+#: src\just-rust.md:36
+msgid ""
+"Like [\\#108905](https://github.com/rust-lang/rust/pull/108905) or "
+"[\\#114613](https://github.com/rust-lang/rust/pull/114613)."
+msgstr ""
+"Como [\\#108905](https://github.com/rust-lang/rust/pull/108905) or "
+"[\\#114613](https://github.com/rust-lang/rust/pull/114613)."
+
+#: src\just-rust.md:41
+msgid "So what's left?"
+msgstr "Entonces, ¿qué queda?"
+
+#: src\just-rust.md:43
+msgid ""
+"Support for the "
+"[LynxOS-178](https://www.lynx.com/products/lynxos-178-do-178c-certified-posix-rtos) "
+"target"
+msgstr ""
+"Soporte para el target"
+"[LynxOS-178](https://www.lynx.com/products/lynxos-178-do-178c-certified-posix-rtos) "
+""
+
+#: src\just-rust.md:44
+msgid "A proprietary safety-critical focussed RTOS"
+msgstr "Un RTOS de seguridad crítica propietario"
+
+#: src\just-rust.md:45
+msgid ""
+"the temporary implementation [of a feature we "
+"proposed](https://github.com/rust-lang/compiler-team/issues/659) and will "
+"implement upstream"
+msgstr ""
+"la implementación de [un feature que "
+"propusimos](https://github.com/rust-lang/compiler-team/issues/659) y que "
+"implementaremos upstream"
+
+#: src\just-rust.md:46
+msgid "Our build and test infrastructure"
+msgstr "Nuestra infraestructura de build y test"
+
+#: src\just-rust.md:51
+msgid "Continuous Test"
+msgstr "Pruebas continuas"
+
+#: src\just-rust.md:53
+msgid "We developed our own _Continuous Integration_ infrastructure"
+msgstr "Desarrollamos nuestra propia infraestructura de _Integración Continua_"
+
+#: src\just-rust.md:54
+msgid "Separate and parallel to that used by The Rust Project"
+msgstr "Distinto y paralelo al usado por el Proyecto de Rust"
```

Comment:
I think it should be "distinta y paralela" as it is talking about the "intregración continua"

Reply:
Good catch!